### PR TITLE
ET-5938 SupplyChain Object Support

### PR DESCRIPTION
**Pull request by mpashka**
## Type of change
- [X] Other

## Description of change
PulsePoint adapter supply chain support

### CODE REVIEW

File: modules/pulsepointBidAdapter.js [hunk 1]
```diff
@@ -428,6 +429,16 @@ function regs(bidderRequest) {
   return null;
 }
 
+/**
+ * Creates source object with supply chain
+ */
+function source(supplyChain) {
+  if (supplyChain) {
+    return {ext: {schain: supplyChain}};
```
Comment (anand-venkatraman) [inline modules/pulsepointBidAdapter.js]: format this properly.
```
{
  ext: {
    schain: supplyChain
  }
}
```

Comment (anand-venkatraman) [reply modules/pulsepointBidAdapter.js]: if you rename the parameter as `schain`, you should be able to do ``` ext: { schain } ``` (ES6)

Comment (mpashka) [reply modules/pulsepointBidAdapter.js]: I've tried to format it but probably original one was better. Also few lines above there is one-liner object creation for gdpr. Probably it's better to leave as is but add spaces around brackets to be consistent.

Comment (anand-venkatraman) [reply modules/pulsepointBidAdapter.js]: I am going to reformat and then apply my change. There is some formatting that is already followed in the file, and I want it keep it that way.

**Pull request closed without merging**